nexus/src/db/datastore.rs

```diff
@@ -430,17 +462,21 @@ impl DataStore {
     /// Updates a organization by name (clobbering update -- no etag)
     pub async fn organization_update(
         &self,
+        opctx: &OpContext,
         name: &Name,
         updates: OrganizationUpdate,
     ) -> UpdateResult<Organization> {
         use db::schema::organization::dsl;
 
+        let (authz_org, _) = self.organization_lookup_noauthz(name).await?;
```

Comment:
Should we use `organization_lookup_id` here?

I know it's functionally the same, but it seems like a good pattern to minimize the number of calls to `organization_lookup_noauthz` until after the auth check is done - at which point, we could just call `organization_lookup` anyway.

Reply:
Yes -- good call!  And nice catch.  Fixed in #616.